index.js

```diff
@@ -1000000,6 +1000000,7 @@ function isEven(number) {
     else if(number === 999999) return false;
     else if(number === 1000000) return true;
     else if(number === 1000001) return false;
+    else return isEven(number - 2);
```

Comment:
I think it would be "safer" to do:
`else return !isEven(number - 1);`

Reply:
I don't believe in using advanced operators like `!`

Reply:
You are absolutely right, it should've been:
```
else return isEven(number - 1) ? false : true;
```

Reply:
This code also implements this part of the project description:

> Hopefully, someday it will be completed... 

and

> PLEASE, MAKE A PULL REQUEST SO WE CAN FINISH THIS PROJECT ASAP.

So I think this is an excellent PR. This code has a possible bug with unbounded recursion if passed a number less than 1, though. (At least until https://github.com/samuelmarina/is-even/pull/71 is merged.)

So I think the ideal would be something like:

```js
else if(isPositive(number) ? return isEven(number - 1) ? false : true : return isEven(number + 1) ? false : true;
```

Unfortunately, I think fixing this is blocked on creating a new `is-positive` library, and I'm not sure if anyone has time to implement that right now.

Reply:
> So I think this is an excellent PR. This code has a possible bug with unbounded recursion if passed a number less than 1, though. (At least until #71 is merged.)
Wouldn't it cause an underflow and eventually work? (assuming the stack size holds up)